src/TextInput/TextInput.features.stories.tsx

```diff
@@ -137,6 +137,7 @@ export const WithTrailingAction = () => {
               }}
               icon={XCircleFillIcon}
               aria-label="Clear input"
+              tooltip-direction="nw"
```

Comment:
Same comment above. Could you please add another story with the `tooltip-direction` passed as a prop rather than editing the Trailing Action story here?